### PR TITLE
3dpandme mods

### PR DESCRIPTION
**Pull request by dmason1992**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
3dpandme:
 - FANnypack zero
 - Thermistor holder zero
#### How was it tested? 
Printed and installed on my printer. See pictures in readmes + blog post on the [topic](https://3dpandme.com/?p=406)
#### Any background context you want to provide?
Always had overheating warning on the Pi with the back panel on so I added a fan to cool it.

I wanted to add chamber temp monitoring while I was at it and the thermistor mod i found did fit where I wanted it so i modified it. This could be added as a remix into his section [samwiseg0](../../samwiseg0/extrusion_thermistor_mount/README.md) but I was unsure of the protocol for that. Happy for it to be moved if deemed more appropriate

### CODE REVIEW

**Review by clee — CHANGES_REQUESTED** [review]
printer_mods/3dpandme/Fannypack_zero/fannypack_zero.stl needs to be rotated so that the STL is in print orientation.

Otherwise this looks good to me.

**Comment by dmason1992** [comment]
> printer_mods/3dpandme/Fannypack_zero/fannypack_zero.stl needs to be rotated so that the STL is in print orientation.
> 
> Otherwise this looks good to me.

Part has be orientated for printing now :)

File: printer_mods/3dpandme/Fannypack_zero/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- 1x modified electronics back panel 
+- 1x printed fan cover
+
+use the dxf and a laser cutter at your local makerspace to produce the holes in the panel or print it out and use it as a template to do it manually. If enough people express interest Funnsor said they would stock some panels so drop me a message if you would want to buy in a whole new panel instead of modifying your old one.
+
+Install m3 inserts in the printed fan cover
+
+Poke wire through the hole and then attach fan and cover to the outside of the panel using the m3 screws.
+
+depending on your
```
Comment (cbc02009) [inline printer_mods/3dpandme/Fannypack_zero/README.md:24]: missing something here :)

File: printer_mods/README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ like so:
 | --- | --- | --- | --- | --- | --- | --- | --- |
 |120decibell|[Side entry exhaust housing](./120decibell/exhaust_housing_side_entry)| Exhaust housing with dual cover screws and single or dual side entry filament paths. |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
 ||[Huvud Chain Mount](./120decibell/huvud_chain_mount)| Huvud rear mount for Clockwork or LGX that supports a cable chain. |:x:|:heavy_check_mark:|:grey_question:|:x:|:heavy_check_mark:|
+|3dpandme|[FANnypack zero](./3dpandme/Fannypack_zero/)|Electronics cooling solution for Voron Zero|:heavy_check_mark:| :x: | :x: | :x: | :x:|
```
Comment (cbc02009) [inline printer_mods/README.md:30]: I think you need to remove the trailing / from the mod links.

File: printer_mods/README.md [hunk 3]
```diff
@@ -27,6 +27,8 @@ like so:
...
+| |[Thermistor holder zero](./3dpandme/Thermistor_holder_zero/)|Extended thermistor holder for Voron Zero|:heavy_check_mark:| :x: | :x: | :x: | :x:|
```
Comment (cbc02009) [inline printer_mods/README.md:31]: techinically this mod will work on the trident and v2, but it's up to you if you want to check them off or not.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by cbc02009** [comment]
Also, all affiliate links must be labeled as such, and a non-affiliate link must also be provided.

File: printer_mods/README.md (hunk 3 above)
Comment (dmason1992) [reply printer_mods/README.md]: Changes done as suggested :)

File: printer_mods/3dpandme/Fannypack_zero/README.md (hunk 1 above)
Comment (dmason1992) [reply printer_mods/3dpandme/Fannypack_zero/README.md]: Done

**Comment by dmason1992** [comment]
> 

Added non affiliate links and made it clear :)

File: printer_mods/3dpandme/Fannypack_zero/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- 1x modified electronics back panel 
+- 1x printed fan cover
+
+use the dxf and a laser cutter at your local makerspace to produce the holes in the panel or print it out and use it as a template to do it manually. If enough people express interest Funnsor said they would stock some panels so drop me a message if you would want to buy in a whole new panel instead of modifying your old one.
+
+Install m3 inserts in the printed fan cover
+
+Poke wire through the hole and then attach fan and cover to the outside of the panel using the m3 screws.
+
+Depending on your control board you may need to use a [klipper expander](https://s.click.aliexpress.com/e/_AfOnF3) board to gain an additional controllable fan output.
```
Comment (cbc02009) [inline printer_mods/3dpandme/Fannypack_zero/README.md:24]: missed an affiliate link here.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/3dpandme/Fannypack_zero/README.md (hunk 4 above)
Comment (dmason1992) [reply printer_mods/3dpandme/Fannypack_zero/README.md]: Oops, pushed the fix now

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
Looks like there's a bad link in the fannypack readme that needs to be fixed.

**Comment by dmason1992** [comment]
> Looks like there's a bad link in the fannypack readme that needs to be fixed.

Think it was the lowercase vs capital I in the installed.jpg image name. Should of fixed it now

**Pull request merged**